_docs/layout/navigation.md

```diff
@@ -233,116 +253,124 @@ navigational element.
 
 # Eyebrow {% include inprogress.html %}
 
-The eyebrow sits as the top most menu bar across all breakpoints within the
-Application Frame of a Helix interface. It provides access to platform level
-services as well as aggregate information across a users accounts at Rackspace.
-The eyebrow pattern is often referred to internally by the project name "Pilot
-Nav" or by the function it serves "Utility Nav".
+The eyebrow is the top menu bar across all breakpoints within the
+application frame of a Helix interface. The eyebrow provides access to
+platform-level services and aggregate information across a user's
+accounts at Rackspace. The eyebrow pattern is often referred to
+internally as "pilot nav" or "utility nav".
 
 ## Breakpoints
 
-The eyebrow is the top most menu bar within a Helix UI. Visual functionality
-expands and retracts based on the current user viewport.
+The eyebrow expands and retracts based on the current user view port.
+
 <div class="row">
 {% column left:"col-md-12" %}
-{% figure [caption:"Eyebrow Breakpoints"] [class:"image bg-light border"] %}
+{% figure [caption:"Eyebrow breakpoints"] [class:"image bg-light border"] %}
 ![]({{site.cdn_url}}/img/layout/eyebrow-states.svg)
 {% endfigure %}
 {% endcolumn %}
 </div>
 
 ## Eyebrow (left side)
-The left side of the eyebrow is comprised of the platform dashboard button and
-product switcher drop-downs. The item currently displayed in the content area
-and navigation section is reflected via the selected state beneath it.
+
+The left side of the eyebrow contains the platform dashboard button
+and the product switcher drop-down menus. The item displayed in the
+content area and navigation section changes based on the selected
+state beneath it.
 
 <div class="row">
 {% column left:"col-md-4" %}
--   The Rackspace® logo serves as a button to trigger the global dashboard.
 
--   Product name to the left of the logo serves as wayfinding to show the
-    currently selected product.
+-   The Rackspace® logo  button triggers the global dashboard.
```

Comment:
The eyebrow has a [fanatiguy icon] + the word [rackspace] in the left corner. this is treated as a single "logo icon". when a user clicks on it, they will trigger the global dashboard to open. 
The product selector is immediately next to the "logo icon" (on the right side)